packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { Authorize } from '@deriv/api-types';
+import { Localize, localize } from '@deriv/translations';
+import { Icon, Money } from '@deriv/components';
+import { Text, Accordion, Button } from '@deriv/ui';
+import { ArrayElement } from 'Types';
+import WalletCard from 'Components/wallet';
+import WalletActionButton from 'Components/wallet-action-button';
+import AppsLauncher from 'Components/app-launcher';
+import { observer } from 'mobx-react-lite';
+
+type WalletAccountProps = {
+    account: ArrayElement<Required<Authorize>['account_list']>;
+};
+
+const WalletAccount = observer(({ account }: WalletAccountProps) => {
+    return (
+        <>
+            <Accordion
+                expand_section
+                elevation_type='xs'
+                onToggle={() => {
+                    /**do something */
+                }}
+                type='containerWithBorder'
+                className='wallet-account'
+            >
+                <Icon className='wallet-account__background' icon='IcAppstoreDemoWallet' />
+                <Accordion.Title size='medium'>
+                    <div className='wallet-account__header'>
+                        <div className='wallet-account__details'>
+                            <div className='wallet-account__logo'>
+                                <WalletCard wallet_name={account.is_virtual === 1 ? 'demo' : ''} size='small' />
```

Comment:
@iman-fs We have same condition at line `54` but they differ, I would suggest to extract it into a helper function to keep the logic consistent.

Reply:
done